config/config.go

```diff
@@ -109,6 +109,17 @@ var (
 	DefaultMarathonSDConfig = MarathonSDConfig{
 		RefreshInterval: Duration(30 * time.Second),
 	}
+
+	// The default Kubernetes SD configuration
+	DefaultKubernetesSDConfig = KubernetesSDConfig{
+		NodeMetricsPort: 10255,
+		ScrapeKey:       "prometheus.io/scrape",
```

Comment:
Ah OK so you're saying the better way to handle this is to pass through service annotations as well & use target relabelling to keep/drop based on that. Right I get it now, thanks! I agree this adds much more flexibility & also reduces the complexity of the Kubernetes discovery piece.

So no filtering in the discovery piece, pass it all through & let relabelling drop/keep targets.

Is my understanding correct now?